todo_app/templates/index.html

```diff
@@ -9,10 +9,46 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <table class="table">
+        <h2>Items</h2>
+        <thead>
+          <tr>
+            <th scope="col">Task</th>
+            <th scope="col">Status</th>
+            <th scope="col"></th>
+            <th scope="col"></th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for item in items %}
+          <td>{{ item.title }}</td>
+          <td>{{ item.status }}</td>
+          <td>
+            <form action="/update_task?id={{ item.id }}&title={{ item.title }}&status={{ 'Completed' if item.status == 'Not Started' else 'Not Started' }}" method="post">
+              <button class="btn btn-outline-primary" type="submit">
+                {% if item.status == "Not Started"%}
+                Mark completed
```

Comment:
Again very minor, but I tend to recommend indenting for Jinja as if it were HTML, so e.g. indenting this text, to make parsing the start/end of if statements easy